examples/dapperish/dapper.go

```diff
@@ -7,5 +7,10 @@ import (
 
 // NewTracer returns a new dapperish Tracer instance.
 func NewTracer(processName string) opentracing.Tracer {
-	return standardtracer.New(NewTrivialRecorder(processName))
+	return standardtracer.New(
+		NewTrivialRecorder(processName),
+		&opentracing.ImplementationID{
```

Comment:
you're explicitly letting tracers return `nil` here? Doesn't seem like a bad idea to force tracers to pick a name and version.